### ISSUE TITLE
508-defect-3 [COGNITION, SCREEN READER]: Screen reader users SHOULD know there is additional descriptive information after the start button

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#19739]
# [508-defect-3](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-3)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** _Josh_

## User Story or Problem Statement

As a screen reader user, I expect relevant information be announced to me if it's descriptive of a button.

## Details

There is relevant information below the "Sign in to start..." buttons on the 526ez page that may be missed if the user does not read the entire page or is navigating by headings and tab stops. Although sighted users can easily tie the button and information together, a screen reader user may skip over it unintentionally.

## Acceptance Criteria

- [ ] Descriptive information of button interactions are tied using `aria-describedby`

## Environment

* Operating System: any
* Browser: any
* Screenreading device: any
* Server destination: production

## Steps to Recreate

1. Enter `https://staging.va.gov/disability/file-disability-claim-form-21-526ez/introduction` in browser
2. Complete the wizard following the BDD flow
3. Start your screen reader of choice
4. Tab to the sign in to start button and confirm the instructions below are not announced

## Proposed Solution

Descriptive information can be linked to the button using `aria-describedby`
**Note: this should also apply to the signed in user's "Start" button**

### Before

```html
<button class="usa-button-primary">
  Sign in to start your application
</button>
<p class="vads-u-margin-top--0">
  By clicking the button to start the disability application, you’ll declare your intent to file. This will reserve a potential effective date for when you could start getting benefits. You have 1 year from the day you submit your intent to file to complete your application.
</p>
```

### After

```html
<button aria-describedby="unique-id" class="usa-button-primary">
  Sign in to start your application
</button>
<p id="unique-id" class="vads-u-margin-top--0">
  By clicking the button to start the disability application, you’ll declare your intent to file. This will reserve a potential effective date for when you could start getting benefits. You have 1 year from the day you submit your intent to file to complete your application.
</p>
```

## WCAG or Vendor Guidance (optional)

* [MDN: Using the aria-describedby attribute](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-describedby_attribute)

## Screenshots or Trace Logs

<img width="1156" alt="Screen Shot 2021-02-10 at 10 53 38 AM" src="https://user-images.githubusercontent.com/14154792/107535008-5c1f1100-6b8e-11eb-966c-0586572c427e.png">

**Comment by joshkimux** [comment]
Would love to get @jimuhh 's eye on this. Reviewing this again, I'm questioning if a code change is the right fix for this problem. I have a feeling that this is more of a content/design problem where "declare your intent to file" should be wrapped into the button or more transparently called out (if it's important for the Veteran to know).

Let me know if this isn't as big of a thing as I think it is, trying to be cautious just in case 🙇

**Comment by jimuhh** [comment]
@joshkimux I agree 100% This is important information and we need to show it to all people in a clear, easy to understand form **before** they follow the CTA. The term "intent to file" and its implications are going to be new for most people so both will need some concise, plain-language explanation.
